traverse_test.go

```diff
@@ -46,3 +47,177 @@ func TestNetworksWithInvalidSearchTree(t *testing.T) {
 	assert.NotNil(t, n.Err(), "no error received when traversing an broken search tree")
 	assert.Equal(t, n.Err().Error(), "invalid search tree at 128.128.128.128/32")
 }
+
+type networkTest struct {
+	Network  string
+	Database string
+	Expected []string
+}
+
+var tests = []networkTest{
```

Comment:
Looks good. I notice that we don't have any tests for networks with no records. Could you add a test case for that?

Reply:
Added in 5800f39